### ISSUE TITLE
How can If check current state is play or pause?

### ISSUE THREAD

**Issue by SapneshNaik** [OHoussein/android-material-play-pause-view#2]

**Comment by pcshady** [comment]
@SapneshNaik  hey did you find the solution for this??

**Comment by pcshady** [comment]
@OHoussein please reply

**Comment by SapneshNaik** [comment]
@pcshady  It is unfortunate that the dev doesn't provide any methods to do this. I suggest you to look for other solutions as it's the best in longer term. However I have come up with a work around 
```  
//initialise your button
  private PlayPauseView linuxPlayPause;
  linuxPlayPause = (PlayPauseView) rootView.findViewById(R.id.linuxPlayPause);

```
Use shared preferences to store the state of the button.
On launch the button is paused so pause => false
```
//this runs every time the app is initialized
        sharedPreferences.edit().putBoolean("isPlay", false).apply();
```
Now every time the button is clicked alter the `isPlay` state.
```
linuxPlayPause.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                    //get isPlay value from sharedPreferences.
                    boolean isPlay = sharedPreferences.getBoolean("isPlay", false);
                   //toggle play / pause
                    linuxPlayPause.toggle();
                   //switch isPlay state
                    sharedPreferences.edit().putBoolean("isPlay", !isPlay).apply();

                    if (!sharedPreferences.getBoolean("isPlay", false)) {

                       //paused, do your stuff

                    } else {


                        //play, do your stuff

                        }


                    }


        });
```

**Comment by OHoussein** [comment]
Fixed, please update it to 1.0.1
